### PR TITLE
Added key-features to config.json

### PR DESCRIPTION
**Pull request by hraftery**
Fixes https://github.com/exercism/x86-64-assembly/issues/88

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -244,7 +244,38 @@
...
+    },
+    {
+      "icon": "powerful",
+      "title": "Powerful",
+      "content": "x86-64 powers the most capable desktop CPUs in the world."
+    },
+    {
+      "icon": "documentation",
+      "title": "Well documented",
+      "content": "Owing to its wide-spread and foundational use, every aspect of x86-64 has been exactingly documented."
```
Comment (ErikSchierboom) [inline config.json:276]: This line is exactly one character too long (max is 100). Could you shorten the text?

File: config.json [hunk 2]
```diff
@@ -244,7 +244,38 @@
       "name": "Basics"
     }
   ],
-  "key_features": [],
+  "key_features": [
+    {
+      "icon": "widely-used",
+      "title": "Widely Used",
+      "content": "x86-64 has been the dominant ISA for personal computers since 2010."
```
Comment (ErikSchierboom) [inline config.json:251]: Maybe it's me, but I didn't know what an ISA is. I expect more people to also not know this acronym. Is there perhaps someway to rephrase?

**Review by ErikSchierboom — COMMENTED** [review]
Thanks a ton! I've left two comments.

**Comment by hraftery** [comment]
Great catches. Was a little sleep deprived and missed them. Have added a commit that should resolve them and generally be an improvement.

**Review by ErikSchierboom — APPROVED** [review]

**Comment by ErikSchierboom** [comment]
Great stuff!

**Comment by ErikSchierboom** [comment]
You can see the key features displayed on https://exercism.org/tracks/x86-64-assembly

**Pull request merged**